README.md

```diff
@@ -587,6 +588,15 @@ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
   -o samples/server/petstore/haskell-servant
 ```
 
+### ASP.NET 5 Web API
+
+```
+java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
+  -i http://petstore.swagger.io/v2/swagger.json \
+  -l aspnet5-webapi \
```

Comment:
Yeah, thanks for catching this. I had to abort my first rebase in which I changed this, and missed it the second rebase.